lxd/container_lxc.go

```diff
@@ -1906,17 +1909,17 @@ func (c *containerLXC) deviceStart(deviceName string, rawConfig map[string]strin
 		return nil, fmt.Errorf("Device cannot be started when container is running")
 	}
 
-	runConfig, err := d.Start()
+	runConf, err := d.Start()
```

Comment:
Can you split the renaming of RunConfig to runConf into its own commit, would make review easier.